viz.yaml

```diff
@@ -109,19 +103,23 @@ fetch:
   -
     id: county_boundaries_census
     location: cache/county_boundaries_census.zip
-    reader: zip
+    reader: shp
     fetcher: url
     remoteURL: 'http://www2.census.gov/geo/tiger/GENZ2015/shp/cb_2015_us_county_500k.zip'
     scripts: []
-    # this file includes counties for Northern Mariana Island (STATEFP=='69'), Guam (STATEFP=='66'), and American Samoa (STATEFP=='60')"
-    # and also the territories we do want: Virgin Islands and Puerto Rico"
+    # see https://www.census.gov/geo/maps-data/data/cbf/cbf_counties.html for details.
+    # this file includes counties for places we don't need:
+    #   Northern Mariana Island (STATEFP=='69'), Guam (STATEFP=='66'), and American Samoa (STATEFP=='60')
+    # and also the places we do want:
+    #   Virgin Islands, Puerto Rico, and all 50 states
   -
     id: county_boundaries_topojson
```

Comment:
Was there a reason for keeping this?

Reply:
we still need county_boundaries_topojson for now because we're relying on it for the state and county dictionaries. as soon as we're creating those from county_boundaries_topojson3, we can remove this one. and then take the 3 off of county_boundaries_topojson3.

Reply:
👍